### PR TITLE
Do Not Update An Existing Project With  A New Group (AST-70718)

### PR DESCRIPTION
**Pull request by cx-miryam-foifer**
By submitting a PR to this repository, you agree to the terms within the [Checkmarx Code of Conduct](../docs/code_of_conduct.md). Please see the [contributing guidelines](../docs/contributing.md) for how to create and submit a high-quality PR for this repo.

### Description

>Do Not Update An Existing Project With  A New Group.

### References

> Include supporting link to GitHub Issue/PR number

### Testing

> Describe how this change was tested. Be specific about anything not tested and reasons why. If this solution has unit and/or integration testing, tests should be added for new functionality and existing tests should complete without errors.
>
> Please include any manual steps for testing end-to-end or functionality not covered by unit/integration tests.

### Checklist

- [ ] I have added documentation for new/changed functionality in this PR (if applicable).
- [ ] I have updated the CLI help for new/changed functionality in this PR (if applicable).
- [ ] All active GitHub checks for tests, formatting, and security are passing
- [ ] The correct base branch is being used

### CODE REVIEW

File: internal/services/projects.go [hunk 1]
```diff
@@ -245,26 +245,9 @@ func updateProject(
...
-	if projectGroups != "" {
-		err = UpsertProjectGroupsByUpdateFlow(groupsWrapper, &projModel, projectsWrapper, accessManagementWrapper, projModelResp, projectGroups, projectID, projectName, featureFlagsWrapper)
-		if err != nil {
-			return projectID, err
-		}
-	}
 	return projectID, nil
 }
 
-func UpsertProjectGroupsByUpdateFlow(groupsWrapper wrappers.GroupsWrapper, projModel *wrappers.Project, projectsWrapper wrappers.ProjectsWrapper,
```
Comment (OrShamirCM) [inline internal/services/projects.go:257]: Im not sure I understand what we did here.. let's talk about it

File: internal/services/projects.go [hunk 2]
```diff
@@ -245,26 +245,9 @@ func updateProject(
...
-	accessManagementWrapper wrappers.AccessManagementWrapper, projModelResp *wrappers.ProjectResponseModel,
-	projectGroups string, projectID string, projectName string, featureFlagsWrapper wrappers.FeatureFlagsWrapper) error {
-	groupsMap, groups, groupErr := GetGroupMap(groupsWrapper, projectGroups, projModelResp, featureFlagsWrapper)
-	if groupErr != nil {
-		return groupErr
-	}
-	projModel.Groups = groups
-	return UpsertProjectGroups(projModel, projectsWrapper, accessManagementWrapper, projectID, projectName, featureFlagsWrapper, groupsMap)
```
Comment (OrShamirCM) [inline internal/services/projects.go:265]: I believe we need to remove new applications as well from updates

Comment (miryamfoiferCX) [reply internal/services/projects.go:265]: I have another task for it

File: internal/services/projects.go (hunk 1 above)
Comment (miryamfoiferCX) [reply internal/services/projects.go:257]: we talked about it...

**Review by OrShamirCM — APPROVED** [review]

**Pull request merged**